src/Elastic.Apm/Model/Payload/Span.cs

```diff
@@ -76,17 +79,24 @@ public void CaptureError(string message, string culprit, StackFrame[] frames)
 		private class ContextImpl : IContext
 		{
 			private readonly Lazy<Dictionary<string, string>> _tags = new Lazy<Dictionary<string, string>>();
+			private readonly Lazy<Dictionary<string, object>> _custom = new Lazy<Dictionary<string, object>>();
 			public IDb Db { get; set; }
 			public IHttp Http { get; set; }
 			public Dictionary<string, string> Tags => _tags.Value;
+			public Dictionary<string, object> Custom => _custom.Value;
 		}
 	}
 
+	public interface IRequest
```

Comment:
Could you please move this into its own file into the `Api` folder? Thx!

Reply:
fixed